features/performance/test/tuned_test.go

```diff
@@ -0,0 +1,61 @@
+package performance_test
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	ini "gopkg.in/ini.v1"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+)
+
+var tunedYamls = []string{
+	"../manifests/00-tuned-network-latency.yaml",
+	"../manifests/01-tuned-rt.yaml",
+}
+
+var _ = Describe("TestPerformanceTuned", func() {
+	table.DescribeTable("Tuned files should provide complete options",
+		func(fileName string) {
+			t := loadTuned(fileName)
+			Expect(t).ToNot(BeNil())
+			validateProfiles(fileName, t)
+		},
+		// msg, fileName
+		table.Entry(tunedYamls[0], tunedYamls[0]),
```

Comment:
doesn´t break anything, but the 2nd arg is unused?

Reply:
this is a sore point actually. The first argument is the title of the test table entry (e.g. "testing that FOO has BAR") the second is the test function argument. Let me try to clarify a bit.

Reply:
ah yes, sure, sorry. Should have known that. /me grabs another :coffee: